src/Domain/Tags.hpp

```diff
@@ -208,6 +209,24 @@ struct Jacobian : db::SimpleTag {
   using type = ::Jacobian<DataVector, Dim, SourceFrame, TargetFrame>;
 };
 
+/// \ingroup ComputationalDomainGroup
+/// Computes the Jacobian of the map held by `MapTag` at the coordinates held by
```

Comment:
Since there is no `MapTag` or `SourceCoordsTag`, I think you should just refer either to the inverse jacobian or the `SourceFrame` and `TargetFrame` template parameters of this template

Reply:
Okay, changed...